action-scheduler.php

```diff
@@ -7,9 +7,9 @@
  * Author URI: https://automattic.com/
  * Version: 3.8.1
  * License: GPLv3
- * Requires at least: 6.2
- * Tested up to: 6.5
- * Requires PHP: 5.6
+ * Requires at least: 6.4
+ * Tested up to: 6.6
+ * Requires PHP: 7.0
```

Comment:
That's a fair leap in php version, though I suppose PHP > 7 would be widely used.

![image](https://github.com/user-attachments/assets/d8c41cad-1211-4062-80c3-9acc2f17f866)
https://stitcher.io/blog/php-version-stats-january-2024

Reply:
Yes, though worth noting that the change in PHP requirements is not new as of this PR—it's just that when we [originally bumped those requirements](https://github.com/woocommerce/action-scheduler/pull/1056) we missed this particular spot (it is however already reflected in the [plugin's `readme.txt`](https://github.com/woocommerce/action-scheduler/blob/3.8.1/readme.txt#L8)).

Also, because of our [L-2 policy](https://developer.woocommerce.com/2023/10/24/action-scheduler-to-adopt-l-2-dependency-version-policy/), our minimum required version of WordPress should now be 6.4 (which itself [does not support PHP 5.6](https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/)).